packages/react-instantsearch-hooks-dom/src/ui/__tests__/Hits.test.tsx

```diff
@@ -0,0 +1,97 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { Hits } from '../Hits';
+
+import type { HitsProps } from '../Hits';
+import type { Hit } from 'instantsearch.js';
+
+describe('Hits', () => {
+  function createProps<THit extends Hit = Hit>(
+    props: Partial<HitsProps<THit>>
```

Comment:
```suggestion
    props: Partial<HitsProps<THit>> = {}
```

Reply:
I'd leave that as-is, no need for variadic call signatures in tests

Reply:
As you prefer.